tslint.json

```diff
@@ -15,6 +15,7 @@
             true,
             "never-prefix"
         ],
+        "max-classes-per-file": [],
```

Comment:
https://github.com/atomist/sdm/blob/master/tslint.json sets this to 7. Do we want to keep that consistent?

Reply:
I think it was defaulting to 1 here and I just quickly added something to fix it.

Yes, we do want to keep it consistent

Reply:
Done